test/test_mbsync.py

```diff
@@ -67,6 +68,62 @@ def test_update_library(self, track_for_mbid, album_for_mbid):
         album.load()
         self.assertEqual(album.album, 'album info')
 
+    @patch('beets.autotag.hooks.album_for_mbid')
+    @patch('beets.autotag.hooks.track_for_mbid')
+    def test_message_when_skipping(self, track_for_mbid, album_for_mbid):
+        album_for_mbid.return_value = \
+            generate_album_info('album id', ['track id'])
+        track_for_mbid.return_value = \
+            generate_track_info('singleton track id',
+                                {'title': 'singleton info'})
```

Comment:
Since you're not querying musicbrainz those patches & return_value are not needed, you can delete them.